src/auth_web.py

```diff
@@ -7,6 +7,7 @@
 import yaml
 import cherrypy
 import requests
+import uuid 
```

Comment:
> Put this on top to fix the Travis CI build.

Imports from the standard library have to appear before other imports. This will fail the build again

```python
import uuid

import yaml
```